lib/Tumblr/StreamBuilder/StreamCursors/StreamCursor.php

```diff
@@ -254,6 +254,10 @@ public static function decode(
                     }
                 } catch (\Exception $e) {
                     $exception = $e;
+
+                    // we might encounter more than one exception, we want to log them all.
```

Comment:
not a huge deal, but because we do throw the last exception, it means that one will be logged _twice_ on our side

Reply:
yep, i'm just saying it cuts both ways 😅